docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -1,12 +1,17 @@
 # Hardware Requirements for EdgeTX
 
+## Support of STM32F MCUs
+* we will not accept new radios based no STM32F2 MCUs<
+* we will not accept new radios with STM32F4 MCUs from EdgeTX v2.12 or EdgeTX v3 onwards (whichever comes first)
+  * except radios that have been announced to the EdgeTX team before that release
+
 ## Color screen radios
 
 ### Mandatory hardware features (color)
 * MCU: presently only STM32F429BI and STM32F439BI with 2 MB flash are supported. Running at 168 MHz.
-  Support for STM32F7/H7 MCUs is planned from EdgeTX v3 onwards, whereas H750 will be the first supported H7 MCU.
+  Support for STM32F7/H7/H7R MCUs is planned from EdgeTX v3 or v2.11 onwards, whereas H750 and H7R will be the first supported H7 MCUs.
```

Comment:
Support for STM32F7/H7/H7R MCUs is planned from EdgeTX v2.11 onwards.

Reply:
F7 should be removed entirely, we will not support those